esmvaltool/preprocessor/_area_pp.py

```diff
@@ -111,7 +122,7 @@ def zonal_means(cube, coordinate, mean_type):
 
 
 # get the area average
-def area_average(cube, coord1, coord2):
+def area_average(cube, coord1, coord2, use_fx_files=False, fx_files=None):
```

Comment:
yes, good point @bouweandela - I had completely missed this code to check; also it's a good idea to check for bounds before collapsing the cube ie @ledm pls reinstate the `cube = _guess_bounds(cube, coords)` before computing the mean at the bottom of your function

Reply:
> reinstate the `cube = _guess_bounds(cube, coords)`

Isn't cmorized data supposed to have bounds in the cube already?

Reply:
ESMValtool in general and preprocessor functions in particular are made to work on cmorized data. If anyone wants to use it on other data, they should take care of cmorizing it before.

Reply:
that's the spirit @bouweandela , I was mucking around with partially CMORized data when I actually put the check on bounds, it should be obsolete by now, agreed. But on the other hand don't underestimate the power of n00bs waltzing in with data that they thinks it's well CMOR :grin:

Reply:
I'm confused as to your conclusions here. Should I add this line back in again or not?

Reply:
don't have to, but I'd do it just in case (I know, a very stable answer from me, right? :rofl: )

Reply:
This line is still there, it's at line 179. 

This function needs the area of each grid cell. It prefers to load it from the fx files. If the fx files are not available, then it calls the `guess_bounds` functions and calculates the gird cell areas. 

Is there still some need to call `guess_bounds` in the fx files section?

Reply:
should be well sorted then, man, cheers!

Reply:
> I think the `use_fx_files` argument is unnecessary: if you provide no fx_files, i.e. None or an empty dict, they cannot be used so you achieve the same.

I've had a go at removing the `use_fx_files` from `_area_pp.py` and the BGC recipe (as reqested by @bouweandela), but it had some strange effects and unusual reactions, like it caused some tests to fail. I think this may well have to wait until a future patch.

Reply:
Can you be more specific? I just tried removing it and the unit tests work just fine, this appears to be untested.